src/components/Card.js

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+import "../scss/components/Card-mobile.scss";
+import "../scss/components/Card-tablet.scss";
+import "../scss/components/Card-desktop.scss";
+
+export default class Card extends React.Component {
+  state = {
+    products: this.props.data.products,
+  };
+
+  selectNormalCard = (index) => {
+    if (
+      this.state.products[index].isActive === false &&
+      this.state.products[index].isError === false
+    ) {
+      this.setState((state) => {
+        return (state.products[index].isActive = true);
+      });
+    } else {
+      this.setState((state) => {
+        return (state.products[index].isActive = false);
+      });
+    }
```

Comment:
To jest mocno nieczytelny kod, można to zapisać znacznie prościej:
```suggestion
    const isActive = this.state.products[index].isActive === false && this.state.products[index].isError === false;
    state.products[index].isActive = isActive;
```